packages/grafana-ui/src/components/Forms/Legacy/Select/Select.story.internal.tsx

```diff
@@ -1,40 +1,63 @@
 import React, { useState, useCallback } from 'react';
 import { action } from '@storybook/addon-actions';
-import { withKnobs, object } from '@storybook/addon-knobs';
+import { Story } from '@storybook/react';
 import { withCenteredStory } from '../../../../utils/storybook/withCenteredStory';
 import { UseState } from '../../../../utils/storybook/UseState';
 import { SelectableValue } from '@grafana/data';
 import { Select, AsyncSelect as AsyncSelectComponent } from './Select';
+import { NOOP_CONTROL } from '../../../../utils/storybook/noopControl';
 
 export default {
   title: 'Forms/Legacy/Select',
   component: Select,
-  decorators: [withCenteredStory, withKnobs],
+  decorators: [withCenteredStory],
+  parameters: {
+    knobs: {
+      disable: true,
+    },
+  },
+  argTypes: {
+    width: { control: { type: 'range', min: 5, max: 30 } },
+    className: NOOP_CONTROL,
+    menuPlacement: NOOP_CONTROL,
+    menuPosition: NOOP_CONTROL,
+    maxMenuHeight: NOOP_CONTROL,
+    minMenuHeight: NOOP_CONTROL,
+    maxVisibleValues: NOOP_CONTROL,
+    prefix: NOOP_CONTROL,
+    renderControl: NOOP_CONTROL,
+    value: NOOP_CONTROL,
+    options: NOOP_CONTROL,
+    tooltipContent: NOOP_CONTROL,
+    components: NOOP_CONTROL,
+    inputValue: NOOP_CONTROL,
+    id: NOOP_CONTROL,
+    inputId: NOOP_CONTROL,
+    defaultValue: NOOP_CONTROL,
+    loading: NOOP_CONTROL,
+    'aria-label': NOOP_CONTROL,
+  },
 };
 
-const initialState: SelectableValue<string> = { label: 'A label', value: 'A value' };
+const value = { label: 'A label', value: 'A value' };
```

Comment:
```suggestion
const initialState: SelectableValue<string> = { label: 'A label', value: 'A value' };
```
Better keep it as `initialState` since that's what it is and also so you don't shadow it in the `onChange` callback. The type is also necessary.

Reply:
Alright.